src/helpers/format/format-ics.ts

```diff
@@ -9,9 +10,9 @@ export function formatIcs(meeting: Meeting) {
 
   if (!meeting.start) return;
 
-  //need an end time. guess one hour if none specified
+  //need an end time. use default if none specified
   if (!meeting.end) {
-    meeting.end = meeting.start.plus({ hour: 1 });
+    meeting.end = meeting.start.plus({ minutes: settings.defaults.duration });
   }
```

Comment:
this is what i was referencing. now that we are supplying `end` times, i think this set of lines can now just be 

```
if (!meeting.start || !meeting.end) return;
```